cob_bringup/drivers/softkinetic.launch

```diff
@@ -6,15 +6,27 @@ The arguments given are the device indices of the cameras determined by the Dept
 -->
 <launch>
 
-	<arg name="name" default="cam3d"/>
+  <arg name="name" default="cam3d"/>
 
-	<node pkg="softkinetic_camera" type="softkinetic_bringup_node" name="$(arg name)" args="0" >
-		<param name="camera_link" type="string" value="/$(arg name)_link" />
-		<param name="confidence_threshold" type="int" value="400" />
-		<param name="use_radius_filter" type="bool" value="False" />
-		<param name="search_radius" type="double" value="0.1" />
-		<param name="minNeighboursInRadius" type="int" value="3" />
-	</node>
+  <node pkg="softkinetic_camera" type="softkinetic_bringup_node" name="$(arg name)" args="0" >
+    <param name="camera_link" type="string" value="/$(arg name)_link" />
+    <param name="confidence_threshold" type="int" value="200" />
+    <param name="use_radius_filter" type="bool" value="False" />
+    <param name="rgb_optical_frame" type="string" value="/$(arg name)_rgb_optical_frame" />
+    <param name="depth_optical_frame" type="string" value="/$(arg name)_depth_optical_frame" />
+    <param name="use_radius_outlier_filter" type="bool" value="false" />
+    <param name="use_serial" type="bool" value="false" />
+    <param name="search_radius" type="double" value="0.05" />
+    <param name="min_neighbours" type="int" value="50" />
+    <param name="depth_mode" value="close" />  <!-- close or long -->
+    <param name="depth_frame_format" value="QVGA" /> <!-- ignored -->
+    <param name="depth_frame_rate" value="25" /> <!-- 25, 30, 50, 60 -->
+    <param name="color_compression" value="MJPEG" /> <!-- YUY2, MJPEG -->
+    <param name="color_frame_format" value="WXGA_H" /> <!-- QQVGA, QVGA, VGA, WXGA_H, NHD -->
+    <param name="color_frame_rate" value="25" /> <!-- 25, 30 -->
+  </node>
+
+  <node name="cam3d_down_relay_depth_points" pkg="topic_tools" type="relay" args="/$(arg name)/depth/points /$(arg name)/depth_registered/points" respawn="false" output="screen"/>
```

Comment:
@fmw-hb why do you have to relay from ../depth/points to ../depth_registered/points?